graphql/src/configuration.tsx

```diff
@@ -1,6 +1,6 @@
 import url from 'url'
 const defaultDdfUrl = 'https://localhost:8993'
-const defaultWebappUrl = 'http://localhost:3000'
+const defaultWebappUrl = 'https://localhost:3000'
```

Comment:
Just realized, does the graphql server need to know about the webapp at all? This key should be able to be removed.

Reply:
This is used for the CORS policy. Would it be better to allow from all origins?
https://github.com/connexta/revelio/blob/941149b4f2478fa56e673a11f3a3b9ed6388da88/graphql/src/apollo-server/index.tsx#L64